### ISSUE TITLE
Compilation issues and unsafe dependencies

### ISSUE THREAD

**Issue by Sroose** [rtrompier/google-home-loxone#18]
Hi, I wanted to try out this package, but compilation failed. 
Next to that some dependencies are on old versions that have high security vulnerabilities, does the package not work with the patched versions?

```
npm WARN notice [SECURITY] systeminformation has the following vulnerabilities: 2 moderate. Go here for more details: https://www.npmjs.com/advisories?search=systeminformation&version=4.27.8 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] set-value has the following vulnerability: 1 high. Go here for more details: https://www.npmjs.com/advisories?search=set-value&version=2.0.1 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] node-forge has the following vulnerability: 1 high. Go here for more details: https://www.npmjs.com/advisories?search=node-forge&version=0.8.5 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] mem has the following vulnerability: 1 low. Go here for more details: https://www.npmjs.com/advisories?search=mem&version=1.1.0 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] lodash has the following vulnerability: 1 high. Go here for more details: https://www.npmjs.com/advisories?search=lodash&version=4.17.20 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] y18n has the following vulnerability: 1 high. Go here for more details: https://www.npmjs.com/advisories?search=y18n&version=4.0.0 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] yargs-parser has the following vulnerability: 1 low. Go here for more details: https://www.npmjs.com/advisories?search=yargs-parser&version=7.0.0 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] axios has the following vulnerability: 1 high. Go here for more details: https://www.npmjs.com/advisories?search=axios&version=0.18.1 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] axios has the following vulnerability: 1 high. Go here for more details: https://www.npmjs.com/advisories?search=axios&version=0.19.2 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
/usr/local/bin/google-home-loxone -> /usr/local/lib/node_modules/google-home-loxone/bin/global.js

> mdns@2.7.2 install /usr/local/lib/node_modules/google-home-loxone/node_modules/mdns
> node-gyp rebuild

make: Entering directory '/usr/local/lib/node_modules/google-home-loxone/node_modules/mdns/build'
  CXX(target) Release/obj.target/dns_sd_bindings/src/dns_sd.o
In file included from ../src/dns_sd.cpp:1:
../src/mdns.hpp:32:10: fatal error: dns_sd.h: No such file or directory
 #include <dns_sd.h>
          ^~~~~~~~~~
compilation terminated.
make: *** [dns_sd_bindings.target.mk:165: Release/obj.target/dns_sd_bindings/src/dns_sd.o] Error 1
make: Leaving directory '/usr/local/lib/node_modules/google-home-loxone/node_modules/mdns/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/share/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Linux 5.10.17-v7+
gyp ERR! command "/usr/bin/node" "/usr/bin/node-gyp" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/google-home-loxone/node_modules/mdns
gyp ERR! node -v v10.24.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/google-home-loxone/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm"})
```

**Comment by rtrompier** [comment]
Hi,
I'm gonna check.
Thanks for your feedback

**Comment by rtrompier** [comment]
Security issues seem to come from `actions-on-google` package : https://github.com/actions-on-google/actions-on-google-nodejs/issues/407

I have fixed the compilation issue on develop branch.

**Comment by rtrompier** [comment]
Now should be fixed in the 1.0.7 release ;) 
https://github.com/rtrompier/google-home-loxone/releases/tag/1.0.7
